### ISSUE TITLE
Add command to run specific test cases

### ISSUE THREAD

**Issue by Ashton-W** [gfontenot/vim-xcodebuild#21]
While editing a test it would be great to be able to run it and only it. Or to specify a test class or file to quickly test.

This will be tricky. If you look at facebooks xctool you will find they do this by creating a temporary scheme.

**Comment by gfontenot** [comment]
Yeah, this might be out of scope for the current plan. I think the reasonable way to start would be to add more configuration so that users can specify that they would like to use `xctool` instead of `xcodebuild` as their build tool. We'd need way more than that to get single test or test class or test file support though. We could look to [rspec.vim](https://github.com/thoughtbot/vim-rspec) for inspiration on this.
